### ISSUE TITLE
Unshaky will not open

### ISSUE THREAD

**Issue by stephb711** [aahung/Unshaky#163]
**Expected behavior**
I followed the instructions: Placed Unshaky in Applications and gave permission in Security & Privacy-Accessibility and Users & Groups-Login Items. When I go to Finder and double click Unshaky to open it to configure the settings, nothing happens at all. I'm unable to do anything. Can you help me figure out this issue? I've been trying to fix my two double-typing keys for months now. Thank you. babin.14@hotmail.com

**Desktop (please complete the following information):**
 - macOS version: 10.15.6

**Comment by connorlindsey** [comment]
Does unshaky appear in the top menu bar? It's the left-most icon.
<img width="930" alt="image" src="https://user-images.githubusercontent.com/12838032/91510690-13eb8b00-e89b-11ea-8d6d-ce47a6fba4a0.png">
